00_introduction/22_architecture_nodes.txt

```diff
@@ -0,0 +1,52 @@
+[[Nodes_supported_OS, the list of supported Operating Systems for Nodes]]
+
+=== Specifications for Rudder Nodes
+
+The following operating systems are supported for Rudder Nodes and packages are
+available for these platforms:
+
+GNU/Linux:
+
+* Debian 5 to 7
+
+* RedHat Enterprise Linux (RHEL) / RHEL-like 3 to 7
+
+* Fedora 18
+
+* SuSE Linux Enterprise Server (SLES) 10 SP3, 11 SP1, 11 SP3
+
+* Ubuntu 10.04 LTS (Lucid), 12.04 LTS (Precise), 12.10 (Quantal), 14.04 LTS (Trusty)
+
+Windows:
+
+* <<install-on-windows, Microsoft Windows>> Server 2000, 2003, 2008, 2008 R2, 2012
+
+[TIP]
+
+[[install-on-windows, Install on Microsoft Windows]] 
+
+.Windows Nodes 
+
+====
+
+Installing Rudder on Windows requires the commercial version of CFEngine (CFEngine Enterprise).
+Hence, as a starting point, we suggest that you only use Linux machines. Once
+you are accustomed to Rudder, please contact Normation to obtain a demo version for
+Windows platforms.
+
+====
+
+[WARNING]
+
+.Unsupported Operating Systems 
+
+====
+
+It is possible to use Rudder on other platforms than the ones listed here.
+However, we haven't tested the application on them, and can't currently supply
+any packages for them. Moreover, the Techniques may fail. If you wish to try
+Rudder on other systems, please get in touch with us!
```

Comment:
Can you add a link to http://www.rudder-project.org/foswiki/Development/AgentBuild here please?